models/topics.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+function fetchTopics() {
+  return db.query(`SELECT * FROM topics`).then(({ rows }) => {
+    if (!rows) {
+      return Promise.reject({
+        status: 404,
+        msg: "rows do not exist",
```

Comment:
is it the rows that don't exist or the topics?

you also don't have a test for this, and I _believe_ that this won't work, so a test is a must at this point :)